### DISCUSSION TITLE
Is it possible to use FQDN for rightsubnet= ?

### DISCUSSION THREAD

**Discussion by fijam** [strongswan/strongswan#392] [Q&A] [answered]
When `rightsubnet` is not specified, it is assumed to be `right/32`. If `right` is set with FQDM (e.g. `right=vpn.example.com`) it will be resolved to the IP address (e.g. 160.50.xxx.xxx/32). 

However, if I want to manually specify a subnet in addition to the gateway, I need to provide them as ip addresses (e.g. `rightsubnet=10.8.0.0/24 ,160.50.xxx.xxx/32`). As the IP of my gateway is not fixed, I'd like to specify it as `rightsubnet=10.8.0.0/24, vpn.example.com`. This is not supported. Is there a reason for this? Or am I going about it the wrong way?

**Comment by tobiasbrunner** [comment, marked as answer]
Just use `%dynamic` (i.e. `rightsubnet=10.8.0.0/24,%dynamic`) that will be replaced with the peer's IP address.

**Reply by fijam** [reply]
Cheers, that works.
